lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -361,30 +314,17 @@ sub schedule {
             }
             catch {
                 log_debug("Failed to set worker in scheduling state :( bummer! Reason: $_");
-
-                # If we see this, we are in a really bad state.
-                $failure++
-                  if $hard_busy;
             };
 
         }
         else {
-            log_debug("Failed sending job '"
-                  . $allocated->{id}
-                  . "' to worker '"
-                  . $allocated->{assigned_worker_id} . "' : "
-                  . pp($res));
-            $failure++
-              if $soft_busy;    # We failed dispatching it. We might be under load, but it's not a big issue
+            log_debug("Failed sending job '" . $allocated->{job} . "' to worker '" . $allocated->{worker});
```

Comment:
I'd promote this message to error.

Reply:
it's a little less than an error and a little more than debug - but with shared workers it's business as usual that they get 2 jobs assigned at the same time and refuse to accept it.

Reply:
info then, since it's an expected failure